src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritProjectListUpdater.java

```diff
@@ -220,7 +220,7 @@ private void tryLoadProjectList() {
             }
         }
         try {
-            logger.info("Trying to load project list.");
+            logger.trace("Trying to load project list.");
```

Comment:
Personally I like seeing these log messages in the log. As an admin, there is a sense of comfort when I see these ;)